pysat/utils/stats.py

```diff
@@ -0,0 +1,116 @@
+"""
+pysat.utils.stats - statistical operations in pysat
+=========================================
+
+pysat.coords contains a number of coordinate-transformation
+functions used throughout the pysat package.
+"""
+
+import numpy as np
+
+
+# determine the median in 1 dimension
+def median1D(self, bin_params, bin_label, data_label):
+
```

Comment:
Needs docstring

Reply:
Since an equivalent function is in ssnl, should we remove this function?

Reply:
I say we keep it here for now, create an issue, and put a note in the docstring.

Reply:
adding.